CarRental/CarRental.Web/Models/CarListModel.cs

```diff
@@ -15,4 +16,11 @@ public class CarListModel
 
     [Display(Name = "Licence Plate")]
     public string? LicencePlateNumber { get; set; }
-}
+    public CarListModel FillModel(Car baseModel)
```

Comment:
To chyba sam nieszczęśliwie nadałem nazwą CarListModel, lepsze byłoby CarViewModel 
(poprawka może być ew. przy refactoringu)